### DISCUSSION TITLE
一按strat就直接關掉了

### DISCUSSION THREAD

**Discussion by Ginsaiyan** [NateScarlet/auto-derby#151] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/89129658/129866012-da4614a9-6102-49ec-b3c2-a6d28df975e2.png)
我有自己我有自己進去育成視窗 然後按了SRAT案件 autoderby 就關掉了 留下一個空白的CMD  接著去看debug資料夾都是空的

**Comment by NateScarlet** [comment, marked as answer]
发 launcher.log 和 auto_derby.log

**Reply by Ginsaiyan** [reply]
我的python 裝在anaconda裡面所以我在重新安裝 requirement 等一下下

**Reply by Ginsaiyan** [reply]
後來還是用官方的python 就可以了 只是剛安裝的人好像py 要改python

**Comment by Ginsaiyan** [comment]
你有下載官方的python?

RX78NT1FA ***@***.***>於 2021年9月4日 週六，上午1:23寫道：
